plugins/statistics.koplugin/main.lua

```diff
@@ -1123,6 +1126,15 @@ The max value ensures a page you stay on for a long time (because you fell aslee
                             UIManager:show(durations_widget)
                         end,
                         keep_menu_open = true,
+                    },
+                    {
+                        text = _("Freeze finished books statistics"),
+                        checked_func = function() return self.settings.freeze_finished_books end,
+                        callback = function()
+                            self.is_doc_not_freezed = self.is_doc
+                                and (self.is_doc_not_finished or self.settings.freeze_finished_books)
+                            self.settings.freeze_finished_books = not self.settings.freeze_finished_books
```

Comment:
Would read easier if you swap the order of these 2 lines (currently, the first line use the previous value of self.settings.freeze_finished_books - I was going to review and say "... or **not** self.settings.freeze_finished_books")

Reply:
That was the first attempt, then swapped to save one `not`.

Reply:
Clarity/readability/obviousness is better than convoluted optimisation :)